test/test_group.cpp

```diff
@@ -903,8 +903,8 @@ TEST(Group_TableAccessorLeftBehind)
     TableRef table;
     TableRef subtable;
     {
-        Group group;
-        table = group.add_table("test");
+        auto group = std::make_unique<Group>();
```

Comment:
Just out of curiosity: how does this change behavior?

Reply:
valgrind can report invalid accesses to memory that has been freed, but not to stack variables that have gone out of scope within the current function.